### PR TITLE
guides: fix gemini multimodal live link

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

**Comment by markbackman** [comment]
Let's leave that link as is. This is linked from elsewhere.

We need to update the guides/introduction.mdx page with:
```
href="/guides/features/gemini-multimodal-live"
```

File: guides/introduction.mdx [hunk 1]
```diff
@@ -7,6 +7,42 @@ These guides cover key aspects of building and deploying Pipecat applications. C
 
 ## Features
 
+<CardGroup cols={2}>
```
Comment (markbackman) [inline guides/introduction.mdx:10]: This is duplicating the Features section

**Comment by aconchillo** [comment]
> Let's leave that link as is. This is linked from elsewhere.
> 
> We need to update the guides/introduction.mdx page with:
> 
> ```
> href="/guides/features/gemini-multimodal-live"
> ```

Done

File: guides/introduction.mdx (hunk 1 above)
Comment (aconchillo) [reply guides/introduction.mdx]: Duh, sorry, done. I was going to add the AI models section.

**Review by markbackman — APPROVED** [review]
lgtm! Thanks for catching this.

**Pull request merged**